File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/voice_speaker_generation_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class VoiceSpeakerGenerationForm:
    """语音说话人生成请求"""
    title: str = None
    prompt: str
    negative_prompt: str = None
    model: str
    channel: str = None
    reference_assets: List[AssetMediaResource] = None
    n: int = None
    seed: int = None
    async: bool = None
    callback_url: str = None
    extra_params: Dict[str, Any] = None
    biz_scene: str = None
    biz_id: int = None
    conversation_id: int = None
    message_id: int = None
    speaker_name: str = None
    text: str
    speaker_id: str = None
    language: str = None
    speed: float = None
    pitch: float = None
    volume: int = None
    format: str = None
    emotion: str = None
    type: str = None
    reference_asset_count: int = None

File: sdkwork-app-sdk-go/types/voice_speaker_generation_form.go
package types

// 语音说话人生成请求
type VoiceSpeakerGenerationForm struct {
	Title string `json:"title"`
	Prompt string `json:"prompt"`
	NegativePrompt string `json:"negativePrompt"`
	Model string `json:"model"`
	Channel string `json:"channel"`
	ReferenceAssets []AssetMediaResource `json:"referenceAssets"`
	N int `json:"n"`
	Seed int `json:"seed"`
	Async bool `json:"async"`
	CallbackUrl string `json:"callbackUrl"`
	ExtraParams map[string]interface{} `json:"extraParams"`
	BizScene string `json:"bizScene"`
	BizId int `json:"bizId"`
	ConversationId int `json:"conversationId"`
	MessageId int `json:"messageId"`
	SpeakerName string `json:"speakerName"`
	Text string `json:"text"`
	SpeakerId string `json:"speakerId"`
	Language string `json:"language"`
	Speed float64 `json:"speed"`
	Pitch float64 `json:"pitch"`
	Volume int `json:"volume"`
	Format string `json:"format"`
	Emotion string `json:"emotion"`
	Type string `json:"type"`
	ReferenceAssetCount int `json:"referenceAssetCount"`
}
